frontend/projects/ui/src/app/modals/os-welcome/os-welcome.page.html

```diff
@@ -26,12 +26,12 @@ <h4>0.3.4.4</h4>
   </p>
   <h6>Highlights</h6>
   <ul class="spaced-list">
-    <li>Enhanced performance for page load times</li>
-    <li>Improved progress reporting for data transfer to new data drives</li>
-    <li>Enabled use of IP address for local network backups</li>
-    <li>Added change password flow to the UI</li>
-    <li>Disabled LUKS for new Raspberry Pi setups</li>
-    <li>Fixed miscellaneous bugs</li>
+    <li>Https over Tor for faster UI loading times</li>
+    <li>Change password through UI</li>
+    <li>Use IP address for Network Folder backups</li>
+    <li>
+      Multiple bug fixes, performance enhancements, and other small features
+    </li>
```

Comment:
grammar + tense should be the same throughout

```suggestion
    <li>Enable https over Tor for faster UI loading times</li>
    <li>Change password through UI</li>
    <li>Use IP address for Network Folder backups</li>
    <li>
     Fix multiple bugs, enhance performance, and add other small features
    </li>
```